### PR TITLE
Setup Github Actions to run tests against any new PR that is raised

### LINKED ISSUE #32: Setup Continuos Integration (CI)
Now, since we have more contributors joining in we need a mechanism to make sure any new changes that are getting merged is not causing regression.

Setup [Github Actions](https://github.com/features/actions) to run our tests against any new PR that is raised.

### PR DESCRIPTION
**Pull request by Rohan-cod**
This commit fixes #32

### CODE REVIEW

File: .github/workflows/main.yml [hunk 1]
```diff
@@ -0,0 +1,34 @@
+# This is a basic workflow to help you get started with Actions
```
Comment (srgothi92) [inline .github/workflows/main.yml:1]: Can you please remove template comments.

File: .github/workflows/main.yml [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+    # Steps represent a sequence of tasks that will be executed as part of the job
+    steps:
+      # Checks-out your repository under $GITHUB_WORKSPACE, so your job can access it
+      - uses: actions/checkout@v2
+      - name: Install pip
+        run: python -m pip install --upgrade pip
+      - name: Install cython wheel
+        run: pip install cython wheel
+      - name: Install vcf2fhir
+        run: pip install vcf2fhir
```
Comment (srgothi92) [inline .github/workflows/main.yml:32]: Was there a reason to install our vcf2fhir package from pip ? We would like to run tests on the checked out PR or merged changes and not again the published vcf2fhir package. Can you try raising a PR( containing a single failing test) against your own fork and validate it fails the CI or any other way to make sure this is working.

**Review by srgothi92 — COMMENTED** [review]
Nice ! Always add some description of change in the commit message. It helps in understanding the purpose of change when you  are looking at git history. You would not prefer to open Github Issues just to get overview of commits.

**Comment by srgothi92** [comment]
> Nice ! Always add some description of change in the commit message. It helps in understanding the purpose of change when you are looking at git history. You would not prefer to open Github Issues just to get overview of commits.

Sorry my bad, Ignore this comment. I did not see the commit message.

**Comment by Rohan-cod** [comment]
@srgothi92, I deliberately made some changes to the code that will cause the test cases to fail and tested it with a pull request. I have attached a screenshot of the result for reference.
unittest using GitHub Actions

**Review by srgothi92 — COMMENTED** [review]
Nice 🥤 , Keep it up !
Thanks for making this quick changes and validating them. Similar to comment in other PR can you please squash changes in one commit.

**Comment by Rohan-cod** [comment]
@srgothi92, I have squashed the changes into one commit.

**Review by srgothi92 — APPROVED** [review]

**Pull request merged**